docs/documentation/scan.md

```diff
@@ -7,129 +7,115 @@ nav_order: 5
+
+**4** - A **column** identifies a SQL column in your table. Use column names to configure tests against individual columns in the table. A single Soda SQL scan can run many tests in many columns.
+
+**5** - **```id```** and **```feepct```** are column names that identify specific columns in this table. 
+
+**6** - The value for **valid_format** identifies the only form of data in the column that Soda SQL recognizes as valid during a scan. In this case, any row in the ```id``` column that contains data that is UUID format (univerally unique identifier) is valid; anything else is invalid.
+
+**7** - Same as above, only the tests in the ```column``` section of the yaml file run only against the contents of the single, identified column. In this case, the test "invalid_percentage == 0" checks to see if all rows in the ```id``` column contain data in a valid format. If the test passes, it means that 0% of the rows contain data that is invalid; if the test fails, it means that more than 0% of the rows contain invalid data, which is data that is in non-UUID format. 
+
+
+# Run a scan
+
+When you run a scan, Soda SQL uses the configurations in your scan yaml file to prepare, then run SQL queries against data in your database. The default tests and metrics Soda SQL configured when it created the yaml file are focused on finding missing, invalid, or unexpected data in your tables.
+
+Each scan requires the following as input:
+- a warehouse yaml, which represents a connection to your SQL engine
+- a scan yaml file, including its path, which contains the defined metric and test instructions that Soda SQL uses to scan tables in your warehouse
+
+Example command: 
+
+```shell
+soda scan warehouse.yml tables/demodata.yml
+```
+
+## Scan output
+
+By default, the output of a Soda SQL scan appears in your command line interface.
+
+```shell
+  | 2.0.0b18
+  | Scanning tables/demodata.yml ...
+  | Soda cloud: dev.sodadata.io
+  | Executing SQL query: 
+SELECT column_name, data_type, is_nullable 
+FROM information_schema.columns 
+WHERE lower(table_name) = 'demodata' 
+  AND table_catalog = 'sodasql' 
+  AND table_schema = 'public'
+  ...
+  | < 200 {}
+  | 54 measurements computed
+  | 3 tests executed
+  | All is good. No tests failed.
+  | Exiting with code 0
+```
+
+In the above example, Soda SQL executed three tests and all the tests passed. The ```Exit code``` indicates the number of failed tests in the scan; in this case, it is zero.
+
+If you have a **Soda Cloud** account and you have connected it to the soda-sql tool in your environment, soda automatically pushes the scan output to your Soda Cloud account. You can log in and view the Monitor Results; each row in the Monitor Results table represents the output of one scan.
+Optional: insert the output into your data orchestration tool to block the pipeline if it encounters bad data (test) or run in parallel to make you aware (monitor)
```

Comment:
I think it is better to day "configured Soda Cloud account in your project" rather than "connected"

Reply:
Hm. Perhaps. I'm tying it to the language I used in the doc for Connect to Soda Cloud, implying that there is something to do on each side of the connection to get it to work. It's a subtle detail. Let me think on it.